config/locales/de.yml

```diff
@@ -781,11 +781,11 @@ de:
         resend_invitation: Einladung erneut verschicken
         resent_invitation: Der Benutzer wurde erneut per E-Mail eingeladen.
         role_hint:
-          admin: Konto Administratoren können Benutzer und Beiträge des eigenen Kontos verwalten. System Admisitratoren haben Vollzugriff auf alle Konten.
-          others: Konto Administratoren können Benutzer und Beiträge verwalten.
+          admin: Konto-Administratoren können Benutzer und Beiträge des eigenen Kontos verwalten. System-Administratoren haben Vollzugriff auf alle Konten.
```

Comment:
Please no direct changes to generated files. These will lead to merge conflicts.